pitop/processing/line_follower/vi

```diff
@@ -0,0 +1,78 @@
+!/usr/bin/env python3
```

Comment:
This appears to be a bad shebang. I also don't think that this file should be here?